.travis/linux.win32.install.sh

```diff
@@ -16,11 +16,13 @@ MINGW_PACKAGES="mingw32-x-sdl mingw32-x-libvorbis mingw32-x-fluidsynth mingw32-x
 export MINGW_PACKAGES
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-. $DIR/linux.win.download.sh win32
+# Maybe find a better fix: see https://github.com/koalaman/shellcheck/wiki/SC1090
+# shellcheck disable=SC1090
+. "$DIR"/linux.win.download.sh win32
 
 PACKAGES="nsis cloog-isl libmpc3 qt4-linguist-tools mingw32 $MINGW_PACKAGES"
 
-sudo apt-get install -y $PACKAGES
+sudo apt-get install -y "$PACKAGES"
```

Comment:
You've probably found this in Travis-CI already, but in this case the packages should explicitly have spaces between them.  You'll want to add your `# shellcheck disable=FOO` in front of this guy too.